lib/ssl/src/ssl.erl

```diff
@@ -304,6 +304,7 @@
                                 {key, key()} |
                                 {keyfile, key_pem()} |
                                 {password, key_password()} |
+                                {password_fun, key_password_fun()} |
```

Comment:
Why invent a new option instead of letting the password be string or a fun as in public_key? I do not think we need yet another option and what if both are specified?  I think we want to have only one option.

Reply:
If it is not a specific ask to the customer then yes we could put in one option. I tested same and it works. Kindly suggest